src/field338.json

```diff
@@ -0,0 +1,59 @@
+[
```

Comment:
Move to field33XUtils and export as constant
`export const f338Variables = [..]`

Reply:
Fixed

Reply:
Should we remove this file when these are in field33XUtils?

Reply:
Fixed